cmd/crdvalidator/internal/crd/crd.go

```diff
@@ -0,0 +1,172 @@
+package crd
```

Comment:
Should we move this to the internal/crd and have the cmd/crdvalidator executable package import that?

Reply:
I have two main reasons that I moved this package into the `cmd/crdvalidator` package:

1. The understanding that eventually this webhook will broken out into its own repository and it requires this package
2. Nothing else but this webhook is using it currently.

With that in mind, I'm fine with moving it back if we feel that something else in Rukpak will eventually use it.